cmd/svcat/README.md

```diff
@@ -139,6 +139,29 @@ Additional parameters and secrets can be provided using the `--param` and `--sec
 --param p1=foo --param p2=bar --secret creds[db]
 ```
 
+You can also provide provision parameters in the form of a JSON string using the `--params` flag:
```

Comment:
Maybe `--params-json` would be better? Or something that implies that the following will be json format?

`--params` seems like it could be `--params='foo=1,bar=2'`

Is there a precedent for this arg name?

Reply:
Sure, that works! I’ll update.

Reply:
I'm not aware of any precedent, and `--params-json` seems fine to me.

Reply:
I would love the enhancement later where `--params` could launch your editor... Or something like that... the way git will pop your editor open if you do `git commit -m`.

That is future awesome...